app/experimenter/docs/adrs/0001-create-new-nimbus-application.md

```diff
@@ -0,0 +1,119 @@
+  - Isolate Nimbus concerns without fear of breaking Normandy experiments
+  - Consistent development and deployment workflow for both old and new applications
+  - Opportunity to rethink core code/architectural/tooling decisions where appropriate
+- Cons
+  - Lose single source of truth
+    - Possibly require people to know which application to refer to depending on experiment type
+    - Possibly create an API proxy in front of both to have a single endpoint for dependent callers
+    - More difficult to correlate experiments/learnings across the two applications
+  - Changes that must apply to both Normandy and Nimbus experiments must be made in two places while they're both being actively maintained
+  - Divergence in tooling/processes over time may create confusion when context switching between the two
+  - Possibly still somewhat constricted by past core design decisions
+  - [Python/Django may not align with FXA tools/processes](https://github.com/mozilla/fxa/blob/main/docs/adr/0020-application-architecture.md)
+
+### Start A New Application in TypeScript/Node
+
+Creating a new web application in TypeScript/Node would allow us to completely restart our thinking about the entire structure of the application which would result in both code and workflow/UI that are purpose built from the ground up to solely address the concerns of Nimbus experiments.
+
+- Pros
+  - Totally greenfield approach allows fresh perspectives to flourish
+  - Stronger sense of collective ownership from all new contributors
+  - Every decision and line of code built specifically to address the needs of Nimbus
+  - TypeScript provides excellent feedback and guarantees which improves development experience
+  - Native async/websocket support allows richer UI experience when coupled with SPA frameworks (like React)
+  - [Fits more cohesively into the FXA team's tooling/processes](https://github.com/mozilla/fxa/blob/main/docs/adr/0020-application-architecture.md)
+  - Logic/types in TS can be easily shared across multiple projects potentially including Desktop/Mobile clients
+  - TS may integrate more smoothly with Rust than Python does to more tightly couple the server and the new Experiment Client
+- Cons
+  - Lose single source of truth
+    - Possibly require people to know which application to refer to depending on experiment type
+    - Possibly create an API proxy in front of both to have a single endpoint for dependent callers
+    - More difficult to correlate experiments/learnings across the two applications
+  - Changes that must apply to both Normandy and Nimbus experiments must be made in two places while they're both being actively maintained, but with the added complexity that one is in Python/Django and the other is in TS/Node which increases the context switching complexity
+  - Node application frameworks are less mature than Django
+  - No Django Admin Panel for easy data access
```

Comment:
Yeah, that is a thing I miss from living in Django a few years ago. A lot of batteries included in the admin panel & ORM stuff. I have to imagine there's something similar in node-land, but also I haven't been on a project using it if there is one.